### ISSUE TITLE
--tuner_backend 选择swift得到的checkpoint转化为peft后推理速度与直接选择peft得到的checkpoint推理速度不一致

### ISSUE THREAD

**Issue by ddf62** [modelscope/ms-swift#2253]
训练模型：Qwen1.5-7b-chat
训练方式：lora
···
swift sft \
    --model_type qwen1half-7b-chat \
    --model_revision master \
    --sft_type lora \
    --tuner_backend peft \
    --template_type qwen \
    --dtype fp16 \
    --output_dir output \
    --dataset  data/minhash_train_1009-copy.jsonl \
    --train_dataset_sample -1 \
    --num_train_epochs 1 \
    --max_length 4096 \
    --check_dataset_strategy warning \
    --lora_rank 8 \
    --lora_alpha 32 \
    --lora_dropout_p 0.05 \
    --lora_target_modules DEFAULT \
    --gradient_checkpointing true \
    --batch_size 1 \
    --weight_decay 0.01 \
    --learning_rate 1e-20 \
    --gradient_accumulation_steps 1 \
    --max_grad_norm 0.5 \
    --warmup_ratio 0.03 \
    --eval_steps 2000 \
    --save_steps 1 \
    --save_total_limit 100 \
    --logging_steps 100 \
    --seed 42 
···
两次训练的区别仅在于：
第一次--tuner_backend选择swift，得到lora的checkpoint后转化为peft格式，进行推理，每个样本平均耗时为0.8s，
第二次--tuner_backend选择peft，得到lora的checkpoint后直接进行推理，每个样本平均耗时为1.1s。
两次推理样本一致。

## 更新
第三次实验--tuner_backend选择peft，训练时开启--deepspeed zero3-offload，得到checkpoint后直接推理，每个样本平均耗时为1.5s，进一步增加

求教🙏

**Comment by z0o0ey** [comment]
Please try again using the latest versions of Swift and VLLM. If the problem persists or you have any new suggestions or comments, please feel free to create a new question.
